instrumentation/redisson-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/redisson/RedissonDbAttributesExtractor.java

```diff
@@ -52,18 +77,8 @@ protected String spanName(
     }
```

Comment:
Let's just return null when size != 1 -- that's what [the spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md#call-level-attributes) says about `db.operation`:

> If the SQL statement has an ambiguous operation, or performs more than one operation, this value may be omitted.